android/sdk/src/main/java/app/organicmaps/sdk/sound/AudioFocusManagerImplLegacy.java

```diff
@@ -0,0 +1,29 @@
+package app.organicmaps.sdk.sound;
+
+import static android.media.AudioManager.AUDIOFOCUS_GAIN_TRANSIENT_MAY_DUCK;
+
+import android.content.Context;
+import android.media.AudioManager;
+import androidx.annotation.NonNull;
+
+final class AudioFocusManagerImplLegacy extends AudioFocusManager
```

Comment:
nit: Is it hard to do in one class instead of two?

Reply:
I don't like how it looks combined in a single class
We can consider using `AudioManagerCompat` https://github.com/organicmaps/organicmaps/pull/7575
But we'll have to include a new library